07_deploy_masters.sh

```diff
@@ -89,6 +89,12 @@ for i in $(seq 0 2); do
   oc --config ocp/auth/kubeconfig wait nodes/master-$i --for condition=ready --timeout=600s
 done
 
-# Wait for clusterversion operator to report availability
-oc --config ocp/auth/kubeconfig wait clusterversion/version --for condition=available --timeout=600s
+# Wait for bootstrap to complete
+while read line; do
+  echo $line;
+  if [ "$(echo ${line} | awk '{print $4}')" == "bootstrap-complete" ]; then
+    break
+  fi
+done < <(oc --config ocp/auth/kubeconfig get events -n kube-system --no-headers -w -o wide)
```

Comment:
You mentioned on slack needing to retry this, could you just omit the `-w` option and call the get events in a loop with grep?

e.g:

```
while ! oc --config ocp/auth/kubeconfig get events -n kube-system --no-headers -o wide | grep -q bootstrap-complete; do
  sleep 10
done

If we need to tolerate failure we can just || true in the conditional?
```

Reply:
Right, lets try that

Reply:
That worked. Squashed commits